Packages/com.unity.multiplayer.samples.coop/Utilities/Net/ClientAuthority/ClientNetworkTransform.cs

```diff
@@ -0,0 +1,46 @@
+using Unity.Netcode.Components;
+using UnityEngine;
+
+namespace Unity.Netcode.Samples
```

Comment:
```suggestion
namespace Unity.Multiplayer.Samples.Utilities.ClientAuthority
```

To build off of the namespace change.